source/c.txt

```diff
@@ -75,6 +76,35 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+Starting with MongoDB 5.0 and C Driver Version 1.18, you can configure your MongoDB client
+instance to use the Versioned API. The Versioned API maintains backwards
+compatibility, which means that you can deploy your app and upgrade your MongoDB
+server without changing any code or updating your driver. To use the
```

Comment:
Could we omit deployment here because it doesn't imply any change to driver nor server version?

I think if we want to provide a conceptual explanation of Versioned API, we should provide all the scenarios in which it helps you maintain compatibility such as changing driver version or server version (not just upgrading).

Happy to discuss over Zoom if you think it would be helpful.